### PR TITLE
[stable/fluent-bit] Added support to run as privileged user in container

### PR DESCRIPTION
**Pull request by DandyDeveloper**
Signed-off-by: Aaron Layfield <aaron.layfield@gmail.com>

#### What this PR does / why we need it:
Fluent-bit input plugins such as [in_kmsg](https://github.com/fluent/fluent-bit/tree/master/plugins/in_kmsg) need to both have access to the volume (via volumeMount) but also need to be running as root to tail the file. 

This PR enabled that ability via a boolean flag in the values. 

#### Which issue this PR fixes
NA

#### Special notes for your reviewer:
This isn't an issue raised from what I can tell, but is essential. I'm experiencing the issue right now. 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by DandyDeveloper** [comment]
@hectorj2f  @edsiper Can you take a quick look at this? Tiny PR.

**Comment by DandyDeveloper** [comment]
@hectorj2f @edsiper Poke.

File: stable/fluent-bit/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluent-bit
-version: 2.4.3
+version: 2.4.4
```
Comment (kfox1111) [inline stable/fluent-bit/Chart.yaml]: Its a new feature, so should probably be 2.5.0. otherwise, looks good to me.

Comment (DandyDeveloper) [reply stable/fluent-bit/Chart.yaml]: Thanks @kfox1111 

Pushed a version update.

File: stable/fluent-bit/values.yaml [hunk 2]
```diff
@@ -42,6 +42,10 @@ trackOffsets: false
 ## Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass
 priorityClassName: ""
 
+## Run the container as a privileged user. Necessary for certain input plugins.
+## daemonSet.podSpec.container.securityContext.privileged
+runAsPrivileged: false
```
Comment (davidkarlsen) [inline stable/fluent-bit/values.yaml:47]: would it be more flexible to just have a `securityContext` map, which can be toYaml'ed into the descriptor? That way you can control all aspects of the sec. context.

Comment (DandyDeveloper) [reply stable/fluent-bit/values.yaml:47]: Yes, that's not a bad idea. I can do that. It just relies on users being more k8 spec friendly.

**Comment by DandyDeveloper** [comment]
Closing. Looks like this was already done a couple months ago.

**Pull request closed without merging**